transmogrifier/helpers.py

```diff
@@ -75,6 +75,62 @@ def parse_date_from_string(
     return None
 
 
+def parse_geodata_string(geodata_string: str, source_record_id: str) -> list[float]:
+    """Get list of values from a formatted geodata string.
+
+    Example:
+     - "ENVELOPE(-111.1, -104.0, 45.0, 40.9)"
+     - "POLYGON((-80 25, -65 18, -64 33, -80 25))"
+
+     Args:
+        geodata_string: Formatted geodata string to parse.
+        source_record_id: The ID of the record containing the string to parse.
+    """
+    geodata_points = []
+    try:
+        raw_geodata_points = geodata_string.split("(")[-1].split(")")[0].split(",")
+        stripped_geodata_points = map(str.strip, raw_geodata_points)
+        geodata_floats = list(map(float, stripped_geodata_points))
```

Comment:
Apologies, kind of reviewed things backwards...

Similar to `parse_solr_date_range_string()`, wonder if this could benefit from regex?

At this time, I'd propose we _only_ attempt to parse `ENVELOPE(...)` patterns, unsure if we've seen any `POLYGON(...)` yet, and we could adjust if and when we do.

Reply:
Haven't found any easy way to clean up this code block with regex so I'll move on for the sake of expediting our full workflow testing

Reply:
Just figured I'd put this here for future reference -- something I tried was:

```
import re
s = "ENVELOPE(-111.1, -104.0, 45.0, 40.9)"
re_geodata_points = re.compile(r'([-]?\d+[.]?\d+)')
re_geodata_points.findall(s)

>>> ['-111.1', '-104.0', '45.0', '40.9']
```

Reply:
It looks like we'll be able to pass the original string (pending some discussions) but this does work!